<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiToolWindow</class>
 <widget class="QMainWindow" name="MidiToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Midi Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="writeMtcCheckBox">
      <property name="text">
       <string>Write MTC</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="writerGroupBox">
      <property name="title">
       <string>Writer:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="writerInfoLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>00:00:00:00 =&gt; 00:00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PhMediaPanel" name="writerMediaPanel" native="true"/>
       </item>
       <item>
        <widget class="QCheckBox" name="writerLoopPlaybackCheckBox">
         <property name="text">
          <string>Loop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="readMtcCheckBox">
      <property name="text">
       <string>Read MTC</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="readerGroupBox">
      <property name="title">
       <string>Reader:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="readerInfoLabel">
         <property name="text">
          <string>0x since 00:00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="readerTimeCodeLabel">
         <property name="font">
          <font>
           <family>Arial Unicode MS</family>
           <pointsize>45</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string notr="true">00:00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="readerFpsLabel">
         <property name="text">
          <string>0 fps</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="delayLabel">
      <property name="text">
       <string>0 ms</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionSet_TC_In"/>
    <addaction name="actionSet_TC_Out"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuControls"/>
  </widget>
  <action name="actionSet_TC_In">
   <property name="text">
    <string>Set TC In...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionSet_TC_Out">
   <property name="text">
    <string>Set TC Out...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhMediaPanel</class>
   <extends>QWidget</extends>
   <header>PhCommonUI/PhMediaPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
